src/test/java/benchmark/BenchmarkUtils.java

```diff
@@ -17,7 +21,12 @@ static String loadResource(String name) {
             if (resource == null) {
                 throw new IllegalArgumentException("missing resource: " + name);
             }
-            return String.join("\n", Files.readLines(new File(resource.toURI()), Charset.defaultCharset()));
+            byte[] bytes;
+            try (InputStream inputStream = resource.openStream()) {
+                // In GraphQL Java version 21 and above, this Guava helper is replaced with Java 9's readAllBytes()
+                bytes = ByteStreams.toByteArray(inputStream);
```

Comment:
Fun one: readAllBytes is only in Java 9 onwards so here's the Guava equivalent. We selectively shade in Guava files (e.g. Immutable objects already in use) so we're alright to use Guava here.